test/fixtures/schema.sql

```diff
@@ -1072,6 +1072,11 @@ create function test.many_inout_params(INOUT num int, INOUT str text, INOUT b bo
   select num, str, b;
 $$ language sql;
 
+CREATE FUNCTION test.variadic_param(VARIADIC v TEXT[]) RETURNS text
+LANGUAGE SQL AS $_$
+  SELECT 'Hi'::text
+$_$;
```

Comment:
More meaningful test what we can point users to:
```suggestion
CREATE FUNCTION test.variadic_param(VARIADIC v TEXT[]) RETURNS text[]
LANGUAGE SQL AS $_$
  SELECT $1
$_$;
```